src/app/admin/borrowers/borrowers.component.ts

```diff
@@ -30,4 +43,42 @@ export class BorrowersComponent implements OnInit {
       });
     });
   }
+
+  createBorrower() {
+    console.log("In the process of creating a new borrower");
+    this.adminService.createBorrower(this.newBorrower).subscribe(res => {
+      console.log("Successfully created a Borrower");
```

Comment:
While `console.log` statements like these are useful in development and debugging, they probably shouldn't be committed, so as to avoid cluttering the log once the feature is working.